geom_mgr/LatLon/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+target_sources(MAPL.geom_mgr PRIVATE
+
+  LonAxis.F90
+  LatAxis.F90
+  LatLonDecomposition.F90
+  LatLonGeomSpec.F90
+  LatLonGeomFactory.F90
+
+)
+
+esma_add_fortran_submodules(
+  TARGET MAPL.geom_mgr
+  SUBDIRECTORY LatLonDecomposition
+  SOURCES new_LatLonDecomposition_basic.F90 new_LatLonDecomposition_petcount.F90
+          new_LatLonDecomposition_topo.F90 get_lon_distribution.F90
```

Comment:
As I mentioned in Teams - the constructors should stay in the module and not be split as submodules.